solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -1921,9 +1921,7 @@ public ScoreMode scoreMode() {
       final TopDocs topDocs;
       final ScoreMode scoreModeUsed;
       if (!MultiThreadedSearcher.allowMT(pf.postFilter, cmd, query)) {
-        if (log.isDebugEnabled()) {
-          log.debug("skipping collector manager");
-        }
+        log.debug("SINGLE THREADED search,skipping collector manager");
```

Comment:
minor: `getDocListAndSetNC` could benefit from similar logging.

Reply:
IMO this is trace level

Reply:
I'm fine with whatever level, but see also https://github.com/apache/solr/pull/2596#pullrequestreview-2201721099